product_specifications/12_xfomv_e/detail.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Deactivation of repository
+
+        Background:
+            Given that a Consortium Lead or Member cannot find a suitable host
+
+        Scenario: Fabrica de-activation
```

Comment:
I think the current flow is repositories are moved to some DataCite member account and labeled as inactive.

Reply:
Ohh I see, let them add that change